cmd/api/src/queries/graph.go

```diff
@@ -664,6 +667,37 @@ func (s *GraphQuery) GetEntityCountResults(ctx context.Context, node *graph.Node
 	return results
 }
 
+func (s *GraphQuery) CountNodesByKind(ctx context.Context, kinds ...graph.Kind) (int64, error) {
+	var (
+		numNodes int64
+	)
```

Comment:
Since this is the only variable, would it be preferable to have it as a single var line?
```suggestion
	var numNodes int64
```